docs/development/settings.rst

```diff
@@ -46,13 +66,6 @@ Default: :djangosetting:`RTD_INTERSPHINX_URL`
 This is the domain that is used to fetch the intersphinx inventory file.
 If not set explicitly this is the ``PRODUCTION_DOMAIN``.
 
-MULTIPLE_APP_SERVERS
```

Comment:
This is still in the code in some places. We should remove it from there as well. 

I think #7044